### ISSUE TITLE
Operator: cleanup old kube resources

### ISSUE THREAD

**Issue by desaintmartin** [kubeflow/kfctl#371]
I've heard the Operator, like kfctl, does not clean up no-longer-wanted resources when upgrading a kubeflow (like a component which has disappeared in the latest version will still exist in the Kubernetes cluster after kubeflow upgrade to latest version).

Could you confirm?

If so, is it planned to add such support in the Operator or will still run using kustomize as its core which does not allow it?

**Comment by adrian555** [comment]
@desaintmartin yes, we are looking at how to support this as part of the upgrade. We will track this issue under #193.
